ctrlf.el

```diff
@@ -871,11 +896,12 @@ Wrap around if necessary."
 
 ;;;;; Main commands
 
-(defun ctrlf-forward (style &optional preserve)
+(defun ctrlf-forward (style &optional preserve arg pos)
```

Comment:
It turned out it _was_ better to pass both `arg` and `pos` here. Since we pass `pos` straight to `ctrlf--start`, wrapper functions don't have to deal with the edge cases of being in or out of a search, which keeps this logic in exactly one place.

Reply:
Okay!